docs/source/authentication.rst

```diff
@@ -32,13 +36,18 @@ We only recommend you to use this authentication if
 * You do not have user access rights for a project or dataset
 * You want to access a project or dataset as part of an automated pipeline without using a specific user account
 
+.. note::
+
+    To setup resource authentication, follow these instructions in the :ref:`authentication:Resource authentication` section below.
+
+
 User authentication
 ==========================
 
 Set up authentication keys - user authentication
 --------------------------------------------------------------------
 Public keys are tied to the user so need to be added to your profile.
-To register a public key, please follow the instructions in :xref:`public_keys` of the |sdk| documentation.
+To register a public key, please follow the instructions in :xref:`public_keys` of the |platform| documentation.
```

Comment:
```suggestion
To **register a public key**, please follow the instructions in :xref:`public_keys` of the |platform| documentation.
```
Maybe parts of it could be bold